internal/product/contract.go

```diff
@@ -31,9 +31,26 @@ func newFromProduct(product *Product) *ProductResponse {
 	}
 }
 
+type PriceResponse struct {
+	ID           string    `json:"id"`
+	Price        float64   `json:"price"`
+	ModifiedDate time.Time `json:"modified_date"`
+	ModifiedBy   string    `json:"modified_by"`
+}
+
+func newFromPrice(price *Price) *PriceResponse {
```

Comment:
I am curious about the naming. What does 'new' mean in 'newFromPrice'?

Reply:
alas! new here means the `new` PriceResponse struct